scrnaseq.Rmd

```diff
@@ -547,11 +557,11 @@ p = PlotMystyle(p, "UMAP, cells coloured by cluster identity", legend_position="
 p
 ```
 
-## Cell Cycle Effect
+## Cell Cycle Effect {.tabset}
 How much do gene expression profiles in your dataset reflect the cell cycle phases the single cells were in? We determine the effects of cell cycle heterogeneity by calculating a score for each cell based on its expression of G2M and S phase markers. Scoring is based on the strategy described in `r knitcitations::citet("10.1126/science.aad0501")`, and human gene names are translated using biomaRt. 
 ```{r cellCycleEffect}
 # Determine cell cycle effect, this time based on default assay "SCT"
-sc = Seurat::CellCycleScoring(object=sc, s.features=genes_s[,2], g2m.features=genes_g2m[,2], set.ident=FALSE)
+sc = CellCycleScoring(object=sc, s.features=genes_s, g2m.features=genes_g2m, set.ident=FALSE, nbin=15)
```

Comment:
nbin could be depending on the number of features?

Reply:
Yes, or the number of cells. I don't know for sure. Do you want me to change something here? I am not sure about this parameter tbh. I would leave it and keep an eye on it.

Reply:
It worked the whole time so it has something to do with the actual dataset right? Then, I would reset it to default, please.

Reply:
It worked until I updated all packages. It worked for a larger dataset, but it does not work with this smaller test dataset, so no, I cannot just reset it in this case. I will need to use another test dataset then, which is ok. Just needs a bit more time.

Reply:
Here is a possible solution: https://github.com/satijalab/seurat/issues/1227

Reply:
Mmh, any particular solution? I already remove cell cycle genes that are not in `rownames(sc)` any more with no success. 

Increase `min_cells=1`, in case sparsity is a problem? I can try that one.

Reply:
My feeling is that this breaks now, since I specified `min_cells=1` for `SCTransform`. This was `5` by default. With `min_cells=3` it runs through. Which value would you agree on? `3` is intuitive, right? I am currently testing `2`.